### ISSUE TITLE
Question about the colmap parameter setting and image resize need to convert the camera pose

### ISSUE THREAD

**Issue by mayunchao1994** [google/dynamic-video-depth#12]
This is very useful work, thanks. I use `colmap automatic_reconstructor --camera_model FULL_OPENCV` to process the dog training set in DAVIS to get the camera pose, then replacing `./datafiles/DAVIS/triangulation/`, other training codes have not changed, but the depth result of each frame has become much worse. How to set the specific parameters of colmap preprocessing? In addition, the image is resized to a small image during training, does the camera pose information obtained by colmap need to be transformed according to resize?

**Comment by ztzhang** [comment]
Hi, resizing will lead to a change to camera intrinsics, which I think is hadled by the preprocessing code. Camera pose accuracy does effect the reconstruction result. Also, regarding to colmap params, FULL_OPENCV means that there's a lens distortion model that's being optimized as well, but our method assumes a pin hole model. Note that you can use the undistorted images as input to bypass this restrictions, which colmap does provide during the process.

**Comment by mayunchao1994** [comment]
Thanks for your reply, I'm checking where my training went wrong. The camera pose results I obtained with the same dog input are very different from the ones you provided. For example, my feature point   is `-12.834286950976896 20.200215152389312 38.02627542642136` in .obj, and yours is `0.391082 0.041740 -1.121706`. I feel that different colmap parameters will not bring such a big difference, are the intrinsics, matrices, and .obj you provided directly obtained by colmap or have they undergone other processing?
